<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoverClock Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar {
    left: 0; 
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="textTab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="fillColorLabel">
         <property name="text">
          <string>Fill color:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="timeFontLabel">
         <property name="text">
          <string>Time font:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="strokeThicknessSpinBox">
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="strokeThicknessLabel">
         <property name="text">
          <string>Stroke thickness:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="strokeColorPicker">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="strokeColorLabel">
         <property name="text">
          <string>Stroke color:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="fillColorPicker">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dateFontLabel">
         <property name="text">
          <string>Date font:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="dateFontPicker">
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="timeFontPicker">
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="appearanceTab">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="verticalPaddingLabel">
         <property name="text">
          <string>Vertical padding:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="timeFormatLabel">
         <property name="text">
          <string>Time format:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="dateFormatLabel">
         <property name="text">
          <string>Date format:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="opacityLabel">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="horizontalPaddingLabel">
         <property name="text">
          <string>Horizontal padding:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="horizontalPaddingSpinBox"/>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="verticalPaddingSpinBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="dateFormatLineEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="timeFormatLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="positionComboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="opacityDoubleSpinBox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>strokeThicknessSpinBox</tabstop>
  <tabstop>opacityDoubleSpinBox</tabstop>
  <tabstop>positionComboBox</tabstop>
  <tabstop>timeFormatLineEdit</tabstop>
  <tabstop>dateFormatLineEdit</tabstop>
  <tabstop>verticalPaddingSpinBox</tabstop>
  <tabstop>horizontalPaddingSpinBox</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
